lib-multisrc/libgroup/src/eu/kanade/tachiyomi/multisrc/libgroup/LibGroupDto.kt

```diff
@@ -245,6 +245,8 @@ class Chapter(
     }
 }
 
+fun List<Chapter>.getBranchCount(): Int? = this.maxOfOrNull { chapter -> chapter.branches.size }
```

Comment:
you are using non null assertion where you use this function. so better to just use `maxOf` instead

Reply:
This function will not execute if chapter list empty.
There is check for that in `chapterListParse`.

So yes, no need for null assertion.